src/scenes/game.py

```diff
@@ -137,5 +173,17 @@ def update(self, current_time, events):
         if dialog is not None:
             dialog.update(current_time, events)
         else:
+            # first, the script generated events
+            for event in self._get_script_events():
+                try:
+                    if event['type'] == 'io_queue':
+                        IoQueue.Instance.on_click()
+                    elif event['type'] == 'process':
+                        Process.Processes[event['pid']].on_click()
+                    elif event['type'] == 'page':
+                        Page.Pages[(event['pid'],event['idx'])].on_click()
+                except Exception as exc: # pylint: disable=broad-exception-caught
```

Comment:
this broad exception catches any problem that may come from each event
(I had a few of indexError and valueError when testing my own script)